specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/ipAllocation.json

```
@@ -0,0 +1,395 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List IpAllocations in resource group": {
+            "$ref": "./examples/IpAllocationListByResourceGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IpAllocationPropertiesFormat": {
+      "properties": {
+        "subnet": {
+          "$ref": "./virtualNetwork.json#/definitions/Subnet",
+          "description": "The Subnet that using the prefix of this IpAllocation resource."
+        },
+        "virtualNetwork": {
+          "$ref": "./virtualNetwork.json#/definitions/VirtualNetwork",
+          "description": "The VirtualNetwork that using the prefix of this IpAllocation resource."
+        },
+        "type": {
+          "$ref": "#/definitions/IpAllocationType",
+          "description": "The type for the IpAllocation."
+        },
+        "prefix": {
+          "type": "string",
+          "description": "The address prefix for the IpAllocation."
+        },
+        "prefixLength": {
+          "type": "integer",
+          "x-nullable": true,
+          "default": 0,
+          "description": "The address prefix length for the IpAllocation."
+        },
+        "prefixType": {
+          "$ref": "./network.json#/definitions/IPVersion",
+          "default": null,
+          "description": "The address prefix Type for the IpAllocation."
+        },
+        "ipamAllocationId": {
+          "type": "string",
+          "description": "The IPAM allocation ID."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "IpAllocation tags."
+        }
+      },
+      "description": "Properties of the IpAllocation."
+    },
+    "IpAllocation": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IpAllocationPropertiesFormat",
+          "description": "Properties of the IpAllocation."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "IpAllocation resource."
+    },
+    "IpAllocationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpAllocation"
+          },
+          "description": "A list of IpAllocation resources in a resource group."
```

```suggestion
          "description": "A list of IpAllocation resources."
```

This definition is also used for subscription list response

---

Thanks, fixed.